dockerd-entrypoint.sh

```diff
@@ -92,16 +92,20 @@ _tls_generate_certs() {
 # no arguments passed
 # or first arg is `-f` or `--some-option`
 if [ "$#" -eq 0 ] || [ "${1#-}" != "$1" ]; then
-	# set DOCKER_HOST to the default "--host" value (for both standard or rootless)
+        # set "dockerSocket" to the default "--host" *unix socket* value (for both standard or rootless)
```

Comment:
Can you please adjust the space indents to tabs throughout this change (matching the rest of the file)? :pray: :heart:

Reply:
Of course!

Reply:
Updated